Arashayo/TabletopHaven
# Your groups view

## Related Issue
https://app.clickup.com/t/86bxfhe0q

## Changes list
Created groups view and groups details

## Testing instructions
1. Run node backend/src/createTables.js
2. Run node backend/src/fillGames.js
3. Run node backend/src/fillUsers.js
4. Run both backend and frontend
5. Log in
6. Change owner_id: in createGroup.js to 51 and run node backend/src/createGroup.js
7. Try adding users to the group and see how it looks


## Screenshots (if appropriate):
